src/main/java/propertyFiles/Bundle.properties

```diff
@@ -3025,7 +3025,18 @@ api.errors.invalidApiToken=Invalid API token.
 api.ldninbox.citation.alert={0},<br><br> The {1} has just been notified that the {2}, <a href=''{3}''>{3}</a>, cites "<a href=''{4}/dataset.xhtml?persistentId={5}''>{6}</a>" in this repository.
 api.ldninbox.citation.subject={0}: A Dataset Citation has been reported!
 
+#Schema Validation
+schema.validation.exception.value.missing=Invalid data for key:{0} typeName:{1}. 'value' missing.
```

Comment:
Overall, I like how the errors look. From the IT test run:

- "Controlled vocabulary for key:fields typeName:language value:'badlang' is not valid."
- "Invalid data for key:fields typeName:title. Fields with 'multiple' set to true must be a list."
- "Invalid data for key:fields typeName:language. Found value as list but 'multiple' is set to false."
- "Compound value datasetContactName must match typeName of the object. Found datasetContactNme"
- "Invalid data for key:datasetContact typeName:datasetContactNotAllowed. Only datasetContactName, datasetContactAffiliation, datasetContactEmail allowed."


However, the double space shown in all these errors bothers me a bit (after `with the following error:`):

```
"message": "The Dataset JSON provided failed validation with the following error:  Controlled...
"message": "The Dataset JSON provided failed validation with the following error:  Invalid data...
```

Can we reduce it down to a single space?